### PR TITLE
Add sidenav component

### PR DESCRIPTION
**Pull request by maya**
This adds the sidenav component to the site. 

This is a part of headers & nav component - breaking it up in it's own PR bc it's a separate component from headers and navigation.

screen shot 2015-08-21 at 3 13 41 pm

screen shot 2015-08-21 at 3 14 01 pm

### CODE REVIEW

File: _components/headers.md [hunk 1]
```diff
@@ -5,8 +5,107 @@ title: Headers & Navigation
...
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/HeaderNav_FullUI_v1-930width.png">
+  
+  <h2 class="usa-heading">Side Nav</h2>
+
+  <div class="usa-grid">
+    <aside class="usa-width-one-fourth">
+      <ul class="usa-sidenav-list">
```
Comment (msecret) [inline _components/headers.md]: Why can't it just be `.usa-sidenav`?

File: assets/_scss/components/_sidenav.scss [hunk 2]
```diff
@@ -0,0 +1,84 @@
+// Side Nav
+
+.usa-sidenav-list {
+  @include unstyled-list();
+  width: 101%;
+
+  li {
+    border-top: 1px solid #757575;
```
Comment (msecret) [inline assets/_scss/components/_sidenav.scss]: This color should be a variable

File: assets/_scss/components/_sidenav.scss [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+      background-color: $color-gray-lightest;
+    }
+
+    &.usa-current-page {
+      border-left: 4px solid $color-primary;
+      padding-left: 1.4rem;
+    }
+  }
+
+  .usa-sidenav-sub_list {
```
Comment (msecret) [inline assets/_scss/components/_sidenav.scss]: I don't think this needs to be nested in `usa-sidenav_list`.

File: _components/headers.md (hunk 1 above)
Comment (maya) [inline _components/headers.md]: Bc it's applied to the list class and we have sublists nested beneath it (children and grandchild). So thought we should mark this out.

File: assets/_scss/components/_sidenav.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      padding-left: 2.8rem;
+    }
+
+    a:hover,
+    .usa-current-page {
+      border: none;
+      padding-left: 2.8rem;
+    }
+
+    .usa-sidenav-sub_list {
```
Comment (msecret) [inline assets/_scss/components/_sidenav.scss]: This is the same selector that this rule is already nested in, right? What is this rule doing?

Comment (maya) [inline assets/_scss/components/_sidenav.scss]: This is the grandchild list, it's nested in 1rem more from the first sub list.

File: assets/_scss/components/_sidenav.scss [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+      }
+
+      a:hover {
+        padding-left: 3.8rem;
+      }
+    }
+  }
+}
+
+.usa-nav-links {
```
Comment (msecret) [inline assets/_scss/components/_sidenav.scss]: Why is this rule needed? I can't find it being used anywhere.

File: assets/_scss/components/_sidenav.scss (hunk 2 above)
Comment (maya) [inline assets/_scss/components/_sidenav.scss]: Done in a91ac42.

File: assets/_scss/components/_sidenav.scss (hunk 3 above)
Comment (maya) [inline assets/_scss/components/_sidenav.scss]: Done in 4de2865.

File: assets/_scss/components/_sidenav.scss (hunk 5 above)
Comment (maya) [inline assets/_scss/components/_sidenav.scss]: Oops, this was brought in on accident from the headers styles. Removed in 134cedf.

**Pull request merged**